k8s/demo/common/dg-docassembly/dg-docassembly-demo.yaml

```diff
@@ -17,11 +17,11 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: dg-docassembly
-    version: 0.0.4
+    version: 0.0.5
```

Comment:
why is this version so far behind?

Reply:
I am not sure if this version number should match the one in the microservice chart. I believe this product chart is not updated as frequently as the other.

Reply:
> why is this version so far behind?

I am not sure if this version number should match the one in the microservice chart. I believe this product chart is not updated as frequently as the other.